healthConfig/settings.py

```diff
@@ -164,3 +156,9 @@
 AWS_DEFAULT_ACL = None
 AWS_S3_VERITY = True
 DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+
+# Pusher Configuration
+PUSHER_APP_ID = "1793983"
```

Comment:
We can add these keys in the environment variables

Reply:
should we set this on aws?

Reply:
We'll set on both aws and travis ci

Reply:
ok! you and Sarthak will set them for us right? I'll just leave keys here

Reply:
PUSHER_APP_ID
PUSHER_KEY	
PUSHER_SECRET

These are the key names that I have sent on travis and aws. We can use these

Reply:
modified